space-cli/cmd/modules/addons/database.go

```diff
@@ -126,7 +126,7 @@ func addDatabase(dbtype, username, password, alias, version string) error {
 	}
 
 	// Get the hosts file
-	hosts, err := txeh.NewHosts(&txeh.HostsConfig{ReadFilePath: utils.GetSpaceCloudHostsFilePath(), WriteFilePath: utils.GetSpaceCloudHostsFilePath()})
+	hosts, err := txeh.NewHosts(&txeh.HostsConfig{ReadFilePath: utils.GetSpaceCloudHostsFilePath("default"), WriteFilePath: utils.GetSpaceCloudHostsFilePath("default")})
```

Comment:
Don't use hardcoded value for "default". Use a constant define it in the utils package.